src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/SQLFunctions.java

```diff
@@ -67,7 +67,8 @@
 
     private static final Set<String> dateFunctions = Sets.newHashSet(
             "date_format", "year", "month_of_year", "week_of_year", "day_of_year", "day_of_month",
-            "day_of_week", "hour_of_day", "minute_of_day", "minute_of_hour", "second_of_minute"
+            "day_of_week", "hour_of_day", "minute_of_day", "minute_of_hour", "second_of_minute", "month", "dayofmonth",
```

Comment:
the naming scheme seems inconsistent between having underscores between words (e.g dayofmonth vs second_of_minute)

Reply:
The ones without underscore are supported by MySQL so we could put them into the scalar function base, while the underscored functions are ES_s. These function names are

Reply:
@davidcui-amzn Thanks! These functions are to support the BI integration.
The ones without underscore are supported by MySQL so we could put them into the scalar function base, while the underscored functions (if not supported by MySQL) are put into ES_Scalar functions.